teams/tikv/team.json

```diff
@@ -74,7 +74,8 @@
         "wshwsh12",
         "Xuanwo",
         "you06",
-        "zyguan"
+        "zyguan",
+        "pingyu"
```

Comment:
Please arrange it according to alphabet order.

Reply:
Here is the output of reviewers.sort():
```
[
        "3pointer",
        "Fullstop000",
        "Jibbow",
        "Mossaka",
        "MrCroxx",
        "Xuanwo",
        "ekexium",
        "fredchenbj",
        "glorv",
        "gozssky",
        "hi-rustin",
        "jayzhan211",
        "lhy1024",
        "longfangsong",
        "lzmhhh123",
        "nolouch",
        "pingyu",
        "rleungx",
        "tier-cap",
        "wjhuang2016",
        "wshwsh12",
        "you06",
        "zyguan"
]
Should I use this as the list?